packages/block-library/src/post-content/edit.js

```diff
@@ -55,19 +57,41 @@ function Placeholder() {
 	);
 }
 
+function RecursionError() {
```

Comment:
Interesting, maybe something that `useNoRecursiveRenders` should also expose?

Reply:
Hm, interesting. Btw, I made it a separate component because otherwise some of the ref-juggling hooks inside `useBlockProps` — specifically, `useScrollIntoView` — throw errors.

> something that `useNoRecursiveRenders` should also expose?

This assumes `apiVersion: 2`, though, because of the use of `useBlockProps`. Is that acceptable?

Reply:
It would be good to confirm why `useScrollIntoView` causes issues, maybe it was called too late, React is very strict about all the rules of hooks? I don't mind having it in its own component though. I mostly curious to confirm it isn't some sort of bug with the hook.